README.md

```diff
@@ -10,4 +10,58 @@ It allows developers to effortlessly spin up and run RollApps, simplifying the
 development process and making the Dymension Hub
 more accessible.
 
-To learn more about roller and how to use it please visit [the docs](https://docs.dymension.xyz/build/roller).
```

Comment:
related to what @mtsitrin mentioned, I think we can simply leave this line as a reference to a basic usage and what it does.